julia/test/unittest/ndarray.jl

```diff
@@ -1547,8 +1547,8 @@ function test_argmin()
          NaN 2 6]
     x = NDArray(A)
 
-    @test copy(argmin(x, dims = 1)) == [1 2 1]
-    @test copy(argmin(x, dims = 2)) == reshape([1, 2], :, 1)
+    @test copy(argmin(x, dims = 1)) == [2 2 1]
```

Comment:
@wkcn as per numpy it should be [1, 2, 1] its argmin. right ?

Reply:
In NumPy, if the tensor contains NaN, it always return the index of NaN.

Reply:
import numpy as np
a = np.array([[1,5,3], [np.nan, 2, 6]])
print(a.argmin(0)) # 1, 1, 0
print(a.argmin(1)) # 0, 0

It always return the index of NaN if NaN exists.

Reply:
```julia
julia> A
2×3 Array{Float64,2}:
   1.0  5.0  3.0
 NaN    2.0  6.0

julia> argmin(A, dims = 1)
1×3 Array{CartesianIndex{2},2}:
 CartesianIndex(2, 1)  CartesianIndex(2, 2)  CartesianIndex(1, 3)
```
note that Julia is 1-based index

Reply:
Yes. argmin/argmax in Julia are consistent with that in NumPy.